rules-reviewed/technology-usage/tests/samples.http/JaxRsWebService.java

```diff
@@ -0,0 +1,51 @@
+package org.windup.examples.technology.http;
```

Comment:
From #262 we try to use the same folders' convention name, i.e. having test data in `tests/data/http/` in this case

Reply:
Interesting, I must have based that on older version of #262, because I followed naming style used in it.

(I see it now, this commit b39a22f676f882d4c202af4d73e03c663246cb59 changed that...)

Reply:
Yeah, I myself followed a previous example that was one of the odd inconsistencies in our current rules. I'll fix those too soon. :)